dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java

```diff
@@ -117,4 +119,23 @@ public static String getServerId(){
 	}
 
 
+    /**
+     * This method returns the absolute root path for assets
+     *
+     * @return the root folder of where assets are stored
+     */
+    public static String getAbsoluteAssetsRootPath() {
+        String realPath = Config.getStringProperty("ASSET_REAL_PATH", null);
+        if (UtilMethods.isSet(realPath) && !realPath.endsWith(File.separator)) {
+            realPath = realPath + File.separator;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer StringBuilder (non-synchronized) or StringBuffer (synchronized) over += for concatenating strings](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4756315)